pkg/api/defaults-kubelet.go

```diff
@@ -117,9 +117,12 @@ func (cs *ContainerService) setKubeletConfig(isUpgrade bool) {
 		defaultKubeletConfig["--tls-cipher-suites"] = TLSStrongCipherSuitesKubelet
 	}
 
-	if common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.16.0-beta.1") {
+	if common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.16.0") {
 		// for enabling metrics-server v0.3.0+
 		defaultKubeletConfig["--authentication-token-webhook"] = "true"
+		if !cs.Properties.IsHostedMasterProfile() { // Skip for AKS until it supports metrics-server v0.3
```

Comment:
@andyzhangx This should protect AKS from getting this change too early.

I think this is ready for merge consideration

Reply:
thanks
/hold cancel